ml-completion/src/main/kotlin/org/rust/ml/RsMLRankingProvider.kt

```diff
@@ -6,10 +6,15 @@
 package org.rust.ml
 
 import com.intellij.internal.ml.catboost.CatBoostJarCompletionModelProvider
+import com.intellij.internal.ml.completion.DecoratingItemsPolicy
 import com.intellij.lang.Language
 import org.rust.lang.RsLanguage
 
 @Suppress("UnstableApiUsage")
 class RsMLRankingProvider : CatBoostJarCompletionModelProvider("Rust", "rust_features", "rust_model") {
     override fun isLanguageSupported(language: Language): Boolean = language == RsLanguage
+    override fun getDecoratingPolicy(): DecoratingItemsPolicy = DecoratingItemsPolicy.Composite(
+        DecoratingItemsPolicy.ByAbsoluteThreshold(3.0),
+        DecoratingItemsPolicy.ByRelativeThreshold(2.5)
```

Comment:
How these values were chosen? As I can see, different languages use different values

Reply:
The threshold values are chosen separately for each ranking model, such that the probability of selecting the top element is high in a large number of sessions. So the first element is decorated with a star icon if it satisfies these thresholds. We should change these values to the new proper ones (obtained from the corresponding experiment analysis) with every model update.